### DISCUSSION TITLE
Ibax

### DISCUSSION THREAD

**Discussion by serapp23** [IBAX-io/go-ibax#1798] [Soru Cevap - Türkçe] [answered]
I like ıbax

**Comment by yolandadadada** [comment, marked as answer]
thanks

**Comment by Akilopanta** [comment]
ibax mobil uygulaması henüz yapılmadımı.

**Comment by Akilopanta** [comment]
test token transferi çok hızlı. şaşırdım. umarım proje takipçilerine güzel kazanımları olur.
